uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/directives/leave-request-popup.js

```diff
@@ -36,7 +36,8 @@ define([
           contactId: '<',
           leaveRequest: '<',
           leaveType: '@',
-          userRole: '@'
+          userRole: '@',
+          selectedContactId: '<'
```

Comment:
We now have so many variables for contact ID
1. `contactId`
2. `selectedContactId`
3. `adminId`

I am not sure if we need all 3, could you please explain how these 3 are different.